libV2/tasks/DiskUsage.js

```diff
@@ -165,6 +234,18 @@ class MonitorDiskUsage extends BaseTask {
                 await this._expireMetrics();
             }
         }
+
+        if (this._hardLimit !== undefined) {
+            const shouldLock = this._checkHardLimit(size, this.nodeId);
+            if (shouldLock) {
+                moduleLogger.error('hard limit exceeded, disabling writes to warp 10', { nodeId: this.nodeId });
+                await this._disableWarp10Updates();
+            } else {
+                moduleLogger.error('usage below hard limit, ensuring writes to warp 10 are enabled',
+                    { nodeId: this.nodeId });
+                await this._enableWarp10Updates();
```

Comment:
Wondering if we should not keep a state if we are currently above the hard limit or not, in order to avoid the overhead of re-enabling updates every time we happen to be below the hard limit (i.e. executing the `DROP DROP 'scality' UPDATEON` code), even if we did not reach the hard limit yet? Not sure I understand the logic, just asking.

Reply:
I thought about this, but decided that a simpler mechanism for the first version was ok as the overhead is minimal